README.md

```diff
@@ -37,8 +42,29 @@ Creates an infinitely looping readable stream from a video.
 | --- | --- | --- |
 | filename | <code>string</code> | path to video |
 | opts.fps | <code>integer</code> |  |
-| opts.width | <code>integer</code> |  |
-| opts.height | <code>integer</code> |  |
+| opts.width | <code>integer</code> | output width |
+| opts.height | <code>integer</code> | output height |
+| opts.cropWidth | <code>integer</code> | crop width (width and height are required). |
+| opts.cropHeight | <code>integer</code> | crop height |
+| opts.cropX | <code>integer</code> | crop x (x and y are optional. If not set, the   default is the center position of the video). |
+| opts.cropY | <code>integer</code> | crop y |
+
+<a name="module_ffmpeg-loop--module.exports..applyCrop"></a>
+
+#### module.exports~applyCrop(ffmpeg) ⇒
```

Comment:
It's kind of weird having this listed in the README as if it's an API method.  It's not actually part of `module.exports` is it?

Reply:
Consider adding `@private`?